app/src/main/java/com/courseproject/tindar/entities/Message.java

```diff
@@ -0,0 +1,63 @@
+package com.courseproject.tindar.entities;
+
+import java.sql.Timestamp;
+
+public class Message {
+    private String messageID;
+    private String text;
+    private Timestamp timestamp;
+    private String sentFrom;
+    private String sentTo;
+    private String conversationId;
+
+    // Getter methods
+    public String getMessageID() {
+        return messageID;
```

Comment:
Can we use getMessageId instead of getMessageID?

for all get methods in this class can we use this.xxx to refer to the instance variable? e.g. return this.messageId instead of return messageId.

Reply:
I'd actually prefer it if we left the method names alone. The latter is a good convention to maintain.